lib/companies/pending_changes.ex

```diff
@@ -175,9 +175,9 @@ defmodule Companies.PendingChanges do
   defp struct_to_string(%Industry{}), do: "industry"
   defp struct_to_string(%Job{}), do: "job"
 
-  defp update_approval(change, note, approval) do
+  defp update_approval(change, note, %{id: reviewer_id} = _reviewer, approval) do
```

Comment:
Why do you also pattern match on an discarded var here? For readability?

Reply:
Yes, to make clear that the reviewer is passed in there.

Reply:
Ok thanks :)